src/client.ts

```diff
@@ -185,6 +189,13 @@ export class Client {
     return this._environmentData;
   }
 
+  /**
+   * Gets the host url prefix for current app
+   */
+  public get hostURL() {
+    return `${this.environmentData.hostRootUrl}/#/${this._assignedRoute}`;
```

Comment:
I think we probably shouldn't assume the host app is using hash-based routing.  It's probably better if we ask the host to incorporate the `/#/` part into the data it sends for `hostRootUrl`.

Rather than a simple getter here, I was thinking of something more like:
```
asHostUrl(clientRoute) {
  return path.join(this.environmentData.hostRootUrl, this._assignedRoute, clientRoute);
}
```
obviously, we don't have `path.join` in the browser, but there are some related functions around handling leading/trailing slashes in `HostRouter.ts` that might be refactored into some common path utilities.

Reply:
Agree, I don't like hardcoded '#' as well. I could try to make the "hostRootUrl" pass the whole url  before the `assignedRoute`